infrastructure/docker/build/docker-compose-combo.yml

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+version: '2'
+
+services:
+  source:
+    image: trafficcontrol_tarball
+    build:
+      dockerfile: infrastructure/docker/build/Dockerfile-source
+      context: ../../..
+    volumes:
+      - ../../..:/trafficcontrol:z
+
+  traffic_monitor_build:
+    image: traffic_monitor_builder
+    build:
+      dockerfile: infrastructure/docker/build/Dockerfile-traffic_monitor-${COS_VERSION}
```

Comment:
Might be a good idea to have a default `COS_VERSION` specified for 7.  Docker variable intepolation should support this.